pkg/util/time_cache.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/tikv/client-go/v2/oracle"
+	pd "github.com/tikv/pd/client"
+)
+
+const pdTimeUpdateInterval = 1 * time.Second
```

Comment:
Could you lower the interval? It can improve accuracy.
```suggestion
const pdTimeUpdateInterval = 200 * time.Millisecond
```